db/migrations/20190320104850_initial.js

```diff
@@ -10,26 +10,26 @@ exports.up = function(knex, Promise) {
 
       table.timestamps(true, true);
     }),
-    knex.schema.createTable('playlists_favorites', function(table){
+    knex.schema.createTable('playlists', (table) => {
       table.increments('id').primary();
-      table.integer('favorite_id').references('favorites.id');
-      table.integer('playlist_id').references('playlists.id');
+      table.string('title');
 
       table.timestamps(true, true);
     }),
-    knex.schema.createTable('playlists', function(table){
```

Comment:
I see you switched these. I imagine it was because the foreign key on the playlists favorites table needed playlists to exist.... ? Probably